<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadPoems</class>
 <widget class="QDialog" name="ReadPoems">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>World of Poetry - Read Poems</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
 color: rgb(0, 32, 63);
background-image: url(:/img/images/Country Birds Grayscale Coloring Book for Adults_.jpeg);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Poetry Feed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_filter">
       <property name="text">
        <string>Filter by:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_filter">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 255, 255)</string>
       </property>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Category</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Author</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Recent</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_category">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);</string>
       </property>
       <item>
        <property name="text">
         <string>Love</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nature</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Life</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Happiness</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sadness</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Friendship</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_refresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget_poems">
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
background-image: url(:/img/images/Country Birds Grayscale Coloring Book for Adults_.jpeg);</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_viewPoem">
       <property name="text">
        <string>View Selected Poem</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="text">
        <string>Back to Home</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
